rules/integrations/aws/persistence_lambda_backdoor_invoke_function_for_any_principal.toml

```
@@ -0,0 +1,104 @@
+[metadata]
+creation_date = "2024/04/30"
+integration = ["aws"]
+maturity = "production"
+min_stack_comments = "AWS integration breaking changes, bumping version to ^2.0.0"
+min_stack_version = "8.9.0"
+updated_date = "2024/05/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies when an AWS Lambda function policy is updated to allow public invocation. This rule specifically looks for
+the `AddPermission` API call with the `Principal` set to `*` which allows any AWS account to invoke the Lambda function.
+Adversaries may abuse this permission to create a backdoor in the Lambda function that allows them to execute arbitrary
+code.
+"""
+false_positives = [
+    "    Lambda function owners may legitimately update the function policy to allow public invocation.\n    ",
+]
+from = "now-60m"
+index = ["filebeat-*", "logs-aws.cloudtrail-*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License v2"
+name = "Allow Public Invocation Added to Lambda Function Policy"
```

```suggestion
name = "AWS Lambda Function Policy Updated to Allow Public Invocation"
```

I think we should be consistent and keep AWS in the rule names